src/main/resources/locales/cy/translation/change-legal-representative.json

```
@@ -0,0 +1,15 @@
+{
+  "title": "[W]Changing legal representative",
+  "pageTitle": "[W]Changing legal representation",
+  "h1": "[W]Do you want to change your legal representative?",
+  "contactTheTribunal": "[W]contact the tribunal",
+  "returnToCaseOverview": "[W]Return to case overview",
+  "legalRepresentativeChange": "[W]Yes, I confirm I want to change my legal representative.",
+  "legalRepresentativeRemove": "[W]Yes, I confirm I wish to remove my legal representative and continue my case representing myself.",
+  "submitBtn": "[W]Submit",
```

Worth checking the codebase as we do already have some in the code like submit
```suggestion
  "submitBtn": "Cyflwyno",
```